Resources/Prototypes/_NF/Catalog/Fills/Backpacks/npc_loot_mercenaries.yml

```
@@ -373,6 +373,8 @@
     # Weapons
     - id: WeaponRifleGestio
       prob: 0.9
+    - id: WeaponRifleMR8 # mono
```

YAMLlint (1.35.1)

[warning] too few spaces before comment

(comments)